openreview/conference/templates/paper_recruitment_process.py

```diff
@@ -102,6 +116,16 @@ def process(client, note, invitation):
             ## - Send email
             response = client.post_message(subject, [edge.tail], message)
 
+            ## Send email to inviter
+            subject=f'[{SHORT_PHRASE}] {REVIEWER_NAME} {preferred_name} accepted to review paper {submission.number}'
+            message =f'''Hi {{{{fullname}}}},
```

Comment:
same here @celestemartinez

Reply:
It's trickier to not repeat the submission number here:
Option#1:

> The reviewer Melisa that you invited has accepted the invitation and is now assigned to paper 5.

Option#2:

> The reviewer Melisa that you invited to review paper 5 has accepted the invitation and is now assigned to paper 5.

Option#3:

> The reviewer you invited, Melisa, has accepted the invitation and is now assigned to paper 5.

what do you think?

Reply:
I like Option #2